src/common/ActionButtons/ActionButton.module.css

```diff
@@ -0,0 +1,21 @@
+.button {
+	cursor: pointer;
+	display: block;
+	font-weight: 600;
+	font-size: 14px;
+	line-height: 160%;
+	font-family: inherit;
+	padding: 10px;
```

Comment:
I think looks better adding a couple of px on left and right.
Something like     `padding: 10px 12px;`

Reply:
We should follow the given designs by the agency, I will compare and try you suggestion